### PR TITLE
chore: update debian base image to v2.1.3

### LINKED ISSUE #221: Update debian base image to v2.1.3
Fixes the following vulnerabilities:

- https://lists.debian.org/debian-security-announce/2020/msg00101.html by bumping libgnutls30 from 3.6.7-4+deb10u3 to 3.6.7-4+deb10u5
- https://lists.debian.org/debian-security-announce/2020/msg00089.html by bumping apt version from 1.8.2 to 1.8.2.1.

Update the base image before the next release as updating `docker/` dir will trigger the image push job. We don't want to build an image currently.

### PR DESCRIPTION
**Pull request by aramase**
**Reason for Change**:

**Requirements**

- [x] squashed commits
- [ ] included documentation
- [ ] added unit tests and e2e tests (if applicable).

**Issue Fixed**:

fixes #221 

**Please answer the following questions with yes/no**:

- Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution? **no**

**Special Notes for Reviewers**:

### CODE REVIEW

**Review by ritazh — APPROVED** [review]
lgtm

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ARG TARGETARCH
 ARG TARGETOS
 
-FROM us.gcr.io/k8s-artifacts-prod/build-image/debian-base-amd64:v2.1.0
```
Comment (sozercan) [inline Dockerfile:4]: I think new format for these is `buster-v1.2.0`

https://github.com/kubernetes/release/pull/1496

Comment (aramase) [reply Dockerfile:4]: Ah, so they are tagging both -
```yaml
    "sha256:dc06e242160076b72bd75135fb3dd0a9e91f386b2d812ec10cbf9e65864c755d": ["buster-v1.1.3", "v2.1.3"]
```

Comment (aramase) [reply Dockerfile:4]: Let me update it to use the new format

Comment (sozercan) [reply Dockerfile:4]: do we want `1.1.3` or `1.2.0`?

Comment (aramase) [reply Dockerfile:4]: `1.1.3` is `v2.1.3`. Looks like `1.2.0` was promoted 3hrs ago - https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/images/k8s-staging-build-image/images.yaml#L14. We can go to `1.2.0`.

Comment (aramase) [reply Dockerfile:4]: Updated to `buster-v1.2.0`

**Review by sozercan — APPROVED** [review]

**Pull request merged**